docs/EDID to Receiver Caps Mapping.md

```diff
@@ -6,49 +6,34 @@
 
-A device which encapsulates a digital video or/and audio signal (usually, from a physical connection) into IP packets. Conversely, the device MAY perform the reverse operation for IP packets to a digital signal.
+The Receiver SHALL include any types of conversion, omit any restrictions, or reflect the internal processing in own Receiver Capabilities.
 
-### Sink
+The term 'Receiver' used in this document is defined in [IS-04][IS-04].
 
-A media consuming unit described by an EDID which can be associated to a Receiver.
+The terms 'Parameter Constraint' and 'Constraint Set' used in this document are defined in [BCP-004-01][BCP-004-01].
 
-## Introduction
+The term 'Output' used in this document is defined in [IS-11][IS-11].
 
-Gateway Devices which bridge a NMOS Receiver to baseband connection need to expose the capabilities of the baseband interface through the NMOS resource to enable connection management. This can be obtained by mapping a Sink's EDID to a Receiver's Capabilities and factoring the abilities of the Gateway Device.
+BCP-005-01 is intended to be used in conjunction with an [IS-11][IS-11] and [BCP-004-01][BCP-004-01] deployment; however it has been written in such a way to provide useful functionality even in the absence of such a system.
 
-The mapping is affected when a Gateway Device is:
+This document is targeted against [E-EDID A2][E-EDID] which consist of EDID 1.4 (and covers EDID 1.3) and is referred to as _Base EDID_ and the [CTA-861-G][CTA-861] Extension Block imposed by HDMI. The information present in the EDID is subject to the requirements of the [Display Monitor Timing (DMT)][DMT] specification Version 1 Rev 12.
 
-- Capable of altering a signal (e.g. it is able to consume 4K and downscale it for the Sink who supports up to HD resolutions)
-- Constrained beyond the capabilities of the Sink (e.g. it supports resolution up to 1080 but is connected to a 4k monitor)
-- Using any internal processing before passing the signal to the Sink (e.g. support multiple Sinks per Receiver and use the greatest common denominator to constrain itself)
+## Use of Normative Language
 
-The Gateway Device SHALL include any types of conversion, omit any restrictions, or reflect the internal processing in the capabilities of the Receiver.
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][RFC-2119].
 
 ## Video Receivers
 
-If an [IS-04][IS-04] Video Receiver is associated with a Sink which has EDID, its supported video formats MUST be mapped into Receiver's Capabilities according to the rules below.
+If an [IS-04][IS-04] Video Receiver is associated with an Output which has EDID, its supported video formats MUST be mapped into Receiver's Capabilities according to the rules below.
```

Comment:
If an [IS-04][IS-04] Video Receiver is associated with an Output which has an EDID, the optional mapping of the EDID supported video formats into Receiver's Capabilities SHALL be performed according to the rules below.

Reply:
```suggestion
If an [IS-04][IS-04] Video Receiver is associated with an Output which has an EDID, the optional mapping of the EDID supported video formats into Receiver's Capabilities SHALL be performed according to the rules below.
```